example.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<body>
+<h1>Publish Subscribe on Encrypted channel example</h1>
+<button id="publish">Publish encrypted messages</button>
+<br><br>
+<textarea id="subscriptions" style="width: 100%" rows="15"></textarea>
+</body>
+
+<!-- Include the latest Ably Library  -->
+<script src="https://cdn.ably.io/lib/ably.min.js"></script>
+
+<!-- Instance the Ably library  -->
+<script type="text/javascript">
+    var apiKey = 'INSERT-YOUR-API-KEY-HERE';
+    var channelName = 'encrypted:messages';
+
+    function createChannel(channelName, channelOpts) {
+        var realtime = new Ably.Realtime(apiKey);
+        return realtime.channels.get(channelName, channelOpts);
+    }
+
+    function randomMessage() {
+        return "Message id (" + Math.random() + ")";
+    }
+
+    Ably.Realtime.Crypto.generateRandomKey(function (err, key) {
+        var cipherParams = Ably.Realtime.Crypto.getDefaultParams({key: key});
```

Comment:
Not sure why you're using the long way here (`getDefaultParams` when you used the short way (`{ cipher: { key: key } }`) in the nodejs tutorial? We should use the short way in both.

Reply:
Ah - sorry for the inconsistency; my intention was to see which one would be preferable in a tutorial.
One has the benefit of being less code and succinct, while other let's the reader know about `Ably.Realtime.Crypto.getDefaultParams()`

I'll make the code consistent by keeping the short version